osf/metrics.py

```diff
@@ -417,47 +419,83 @@ def get_registries_info(cls) -> list:
-                                'match': {
-                                    'to_state': {
-                                        'query': 'accepted'
-                                    }
+                                'bool': {
+                                    'must': [
+                                        {
+                                            'match': {
+                                                'to_state': RegistrationModerationStates.ACCEPTED.db_name
+                                            }
+                                        },
+                                        {
+                                            'match': {
+                                                'trigger': RegistrationModerationTriggers.SUBMIT.db_name
+                                            }
+                                        }
+                                    ]
                                 }
                             }
                         },
                         'rejected': {
                             'filter': {
-                                'match': {
-                                    'trigger': {
-                                        'query': 'reject'
-                                    }
+                                'bool': {
+                                    'must': [
+                                        {
+                                            'match': {
+                                                'to_state': RegistrationModerationStates.REJECTED.db_name
+                                            }
+                                        },
+                                        {
+                                            'match': {
+                                                'trigger': RegistrationModerationTriggers.REJECT_SUBMISSION.db_name
+                                            }
+                                        }
+                                    ]
                                 }
                             }
                         },
                         'withdrawn': {
                             'filter': {
-                                'match': {
-                                    'trigger': {
-                                        'query': 'reject'
-                                    }
+                                'bool': {
+                                    'must': [
+                                        {
+                                            'match': {
+                                                'to_state': RegistrationModerationStates.WITHDRAWN.db_name
+                                            }
+                                        },
+                                        {
+                                            'match': {
+                                                'trigger': RegistrationModerationTriggers.ACCEPT_WITHDRAWAL.db_name
```

Comment:
fyi: cutting this (i.e. just looking at the end state) should lead to this count also including force withdrawn registrations. the requirement is unclear, so go ahead and follow your gut as to which is better